### PR TITLE
fix web cases

### PR DESCRIPTION
**Pull request by SSshahan**

### CODE REVIEW

**Comment by SSshahan** [comment]
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/102265/console

File: lib/openshift/subscription.rb [hunk 1]
```diff
@@ -7,5 +7,13 @@ def current_csv(user: nil, quiet: false, cached: false)
       return rr.dig('status', 'currentCSV')
     end
 
+    def ready?(user:, quiet: false, cached: false)
+       res = get(user: user, quiet: quiet)
```
Comment (pruan-rht) [inline lib/openshift/subscription.rb:11]: @SSshahan, I think you forgot to put `cached: cached` on line # 11

**Comment by SSshahan** [comment]
@pruan-rht  updated, could you help to merge this? thanks!

**Pull request merged**